test/api/application/list_test.go

```diff
@@ -0,0 +1,36 @@
+package application
+
+import (
+	"testing"
+
+	"github.com/konveyor/tackle2-hub/api"
+	"github.com/konveyor/tackle2-hub/test/assert"
+)
+
+func TestApplicationList(t *testing.T) {
+	samples := Samples()
+	// Create.
+	for i := range samples {
+		assert.Must(t, Create(&samples[i])) // Need modify parent array instead of single record created by the loop (to keep created IDs).
+	}
+
+	// Try list.
+	got := []api.Application{}
+	err := Client.Get(api.ApplicationsRoot, &got)
+	if err != nil {
+		t.Errorf("List error: %v", err.Error())
+	}
```

Comment:
This looks like the kind of thing that we don't need a specific `list_test.go` to verify. By this I mean, if you update create and delete test fixtures to list and verify the application is either in (or not in) the list, then we've verified `list` is working without churning through this scenario.

Reply:
I partially agree, but since this is an API test, I'd expect explicit test for each endpoint method.

Reply:
Will be updated merging create with get/list.